src/Sentry.Unity/SentryUnity.cs

```diff
@@ -32,6 +31,7 @@ public static void Init(SentryUnityOptions unitySentryOptions)
             SentryOptionsUtility.SetDefaults(unitySentryOptions);
 
             SentrySdk.Init(unitySentryOptions);
+            SentrySdk.StartSession();
```

Comment:
This approach will be taken by the .NET SDK already out of the box.
It'll introduce a `EnableAutoSessionTracking` that for the time being will be set to `false` by default. But once we release a new major version of the SDK for .NET we'll flip it to `true`.

So here we can set it to `false` before passing the options instance to `SentrySdk.Init` to make sure the .NET SDK is not automatically creating a session when `Init` is called and we're free to use the `MonoBehavior` approach without interference. 

See: https://github.com/getsentry/sentry-dotnet/issues/1054

Reply:
@bitsandfoxes and I discussed on a call and after this auto mode came in, it made sense to have both approaches working side-by-side. 

The .NET SDK will start a session on Init, and the Unity SDK would only be able to start after the first event loop/monobehavior event handlers are invoked so makes sense to have a session already started as early as possible (the .NET default beahvior) and the Unity integration wil make sure not to end it when the first "OnForeground" event kicks in.